spyrk/spark_cloud.py

```diff
@@ -61,7 +62,7 @@ def __init__(self, username_or_access_token, password=None):
         seconds (per device?) to reply as it waits for an answer from the
         disconnected devices.
         """
-        self.spark_api = Hammock('https://api.spark.io')
+        self.spark_api = Hammock('https://api.particle.io')
```

Comment:
Ah sorry about this, I really thought I cherry picked in another branch, this is not related to the expiring cache